### ISSUE TITLE
Add Polyform NC license + Countdown addendum

### ISSUE THREAD

**Issue by erlend-sh** [fairsource/fair.io#31]
### License Name

Polyform NonCommercial v1.0.0 + Countdown

### License Short Identifier

PNCC-1.0.0

### Links

https://polyformproject.org/licenses/countdown/1.0.0/

https://writing.kemitchell.com/2023/12/01/PolyForm-Countdown

https://writing.kemitchell.com/2023/10/24/Scheduled-Relicensing

### Description

@kemitchell might have a different (and canonical) opinion on what the Short Identifier should be, I just winged it.

**Comment by ezekg** [comment]
This license [doesn't seem to be widely used](https://github.com/search?type=code&q=%22PolyForm+Countdown+License+Grant%22), but correct me if I'm wrong there. I'm not super familiar with the Polyform license suite. From a cursory glance, and @chadwhitacre correct me if I'm wrong, I think that a blanket restriction to non-commercial use leans too far from our "minimal restrictions" requirement in the [Fair Source Definition](https://fair.io/about/) and more towards source-available.

**Comment by chadwhitacre** [comment]
> This license [doesn't seem to be widely used](https://github.com/search?type=code&q=%22PolyForm+Countdown+License+Grant%22)

@ezekg Looks like they're planning to use it in https://github.com/commune-os/weird/pull/123.

The README today says that new licenses "should also be adopted at more than one company, at least." Do you have any other companies ready to make the case for recognition with you, @erlend-sh?

**Comment by erlend-sh** [comment]
> Do you have any other companies ready to make the case for recognition with you

Not yet, but now that this discussion has been started I have something I can refer other [Polyform adopters](https://polyformproject.org/adopters/) to.

**Comment by chadwhitacre** [comment]
Sounds good. I'm agreed in principle to dealing in Polyform in some fashion. There's a lot of detail to get through on license specifics vis-a-vis the Fair Source Definition, we'll hash it out together. :)

**Comment by ezekg** [comment]
Agreed. I'd love to explore Polyform more and talk specifics i.r.t. the Fair Source Definition. Like I said earlier, I'm not sure if the Polyform Noncommercial is an exact fit since the restrictions are very wide, but the Polyform Perimeter and Shield variations + Polyform Countdown seem like good mostly-permissive non-competes similar to the minimal restrictions under FSL.

@kemitchell — big fan, feel free to chime in.

**Comment by chadwhitacre** [comment]
Shield is the one @hmeeker @heathermeeker suggested to me the other day.

**Comment by chadwhitacre** [comment]
@erlend-sh If you want to champion this I think it would be good to have a call together to discuss, at least the three of us (you, me, and @ezekg). My email is chadwhitacre@sentry.io if you'd like to initiate that.

**Comment by erlend-sh** [comment]
Thanks for the invite! We’re gonna be busy soft-launching our product in the next two weeks, which should also help reify this particular license combo. So if you don’t mind letting this issue linger for a bit, I’d love to do a call together early next month.

**Comment by chadwhitacre** [comment]
Not a problem, ball is in your court.

Good luck with the launch! :-)

**Comment by chadwhitacre** [comment]
Bump. Any interest in continuing to pursue this, or should we close for now?

**Comment by erlend-sh** [comment]
Fine to close for now 👌
